tests/contrib/sanic/test_sanic.py

```diff
@@ -49,6 +49,11 @@ async def hello(request):
         await random_sleep()
         return json({"hello": "world"})
 
+    @app.route("/hello/<name>")
```

Comment:
:+1: 👍  can we add one more test with multiple parameters?

Reply:
Sure, I'll fix it!

Reply:
@Kyle-Verhoog Updated the existing test

Reply:
awesome! thanks 😄